package.json

```diff
@@ -60,18 +60,18 @@
   },
   "devDependencies": {
     "coveralls": "^3.0.2",
-    "image-diff": "^2.0.0",
     "istanbul": "*",
     "jshint": "^2.9.6",
     "less": "^3.8.1",
+    "looks-same": "^7.2.2",
     "mocha": "",
     "mocha-lcov-reporter": "*",
-    "node-sass": "^4.9.3",
+    "node-sass": "^4.13.0",
```

Comment:
`node-sass` is replaced by `sass` (this could affect css output and tests results)

Reply:
? It looks to me like `node-sass` is still under active development.

Reply:
node-sass is C lib, sass is officially support Dart implementation that is distributed as `sass` package

Reply:
Feel free to provide a seperate pull-request that replaces `node-sass` with `sass`! 
I think this is not in the scope of this PR.